### PR TITLE
Remove problematic naming

### PR DESCRIPTION
**Pull request by tconroy**
Hello! 👋 Hope you are doing well.

This PR refactors internal naming to drop a problematic name in favor of a more neutral one.

I am not a Python developer, and this is my first interaction with this library/codebase, so I apologize if I've failed to follow any particular convention. I found this repo through a discussion on Twitter and wanted to extend a hand to help make this codebase a more inclusive environment. 

Happy to work with the maintainers to find a more suitable name if the one I provided isn't good enough.

Thank you for your time!

### CODE REVIEW

**Comment by dannysepler** [comment]
great PR! just a suggestion, but i think you're missing a few references, for example `IN_<TERM>_PATTERN`

a `git grep --ignore-case <TERM>` could probably help you catch the lingering ones

as far as the test failure, seems like a flake to me? 🤷 maybe it'll resolve itself next commit

**Comment by tconroy** [comment]
Thank you @dannysepler! Great point, I think that will work much better. Will push up a fresh commit this evening when I'm back at my laptop.

**Comment by tconroy** [comment]
Hmm — Not sure why the build is failing. It looks to me like the CI is encountering issues with the `pip install` command?

File: nltk/chunk/regexp.py [hunk 1]
```diff
@@ -1426,7 +1428,8 @@ def demo():
     print("Demonstration of empty grammar:")
 
     cp = chunk.RegexpParser("")
-    print(chunk.accuracy(cp, conll2000.chunked_sents("test.txt", chunk_types=("NP",))))
+    print(chunk.accuracy(cp, conll2000.chunked_sents(
```
Comment (dannysepler) [inline nltk/chunk/regexp.py:1431]: Not the biggest deal, but curious why some formatting changes appeared? I kinda liked this one better before, although each is fairly subjective

**Comment by dannysepler** [comment]
and looks like other open PRs are also failing on the py3.6 install, so i wouldn't worry about it :)

File: nltk/chunk/regexp.py (hunk 1 above)
Comment (stevenbird) [reply nltk/chunk/regexp.py:1431]: :+1: Longer than 80 characters I guess.

**Comment by stevenbird** [comment]
Thanks @tconroy, @dannysepler

**Comment by tconroy** [comment]
Thank you @stevenbird @dannysepler! Cheers, Hope you both have a great 2021.

**Pull request merged**